src/referentiel.js

```diff
@@ -4,14 +4,14 @@ const donneesParDefaut = require('../donneesReferentiel');
 const creeReferentiel = (donneesReferentiel = donneesParDefaut) => {
   let donnees = donneesReferentiel;
 
-  const actionsSaisie = () => donnees.actionsSaisie;
-  const identifiantsActionsSaisie = () => Object.keys(actionsSaisie());
-  const actionSaisie = (id) => actionsSaisie()[id] || {};
-  const positionActionSaisie = (id) => actionSaisie(id).position;
+  const actionsSaisie = (version) => donnees.actionsSaisie[version] || {};
+  const identifiantsActionsSaisie = () => Object.keys(actionsSaisie('v1'));
```

Comment:
⚠️ Ici il y a bien un `'v1'` en dur.   
Pour forcer à toujours utiliser un `id` qui correspond à une action existante en PROD.  
C'est en lien avec le commentaire ci-dessus sur `this.homologation.statutSaisie(this.id)`

Reply:
Peux-tu m'aider à mieux comprendre l'intention de ce bout de code en me donnant un exemple de ce qui irait de travers si on ne forçait pas la version à `'v1'` ?

Reply:
J'ai l'impression que j'en ai assez dit avec les commentaires en-dessous.  
Si je n'ai pas réussi, c'est certainement qu'il faut qu'on procède autrement pour mon explication 😄 .

Reply:
`identifiantsActionSaisie()` prend désormais la `version` en paramètre.